topologies/datacenter-latest/labguides/source/bgp-with-cvp.rst

```diff
@@ -0,0 +1,170 @@
+Layer 3 Leaf-Spine
+==================
+
+.. image:: images/cvp-l3ls/l3ls_1.png
+   :align: center
+   :width: 50 %
+
+.. note:: Did you know the “bgp” script is composed of Python code that
```

Comment:
@jrecchia1029 Remove this note, it is not needed since it is referencing the login.py script when a user would ssh into the topology to configure the lab.